modules/oe_theme_content_organisation/templates/oe-theme-helper-in-page-navigation--node--oe-organisation.html.twig

```diff
@@ -12,7 +12,7 @@
 {% block navigation %}
 <div class="ecl-col-12 ecl-col-lg-3">
   {% if image %}
-  <div class="ecl-u-mb-l">
+  <div class="ecl-u-mb-l figure-transparent">
```

Comment:
Instead of doing this try to pass the property `extra_classes` (string) to the `@ecl-twig/media-container` component and add an ECL utility to either make the background transparent or, if non available, white.

Reply:
white background was the only one available